File: Easy/Java/1252. Cells with Odd Values in a Matrix.java
class Solution {
    public int oddCells(int m, int n, int[][] indices) {
        int[] row = new int[m];
        int[] col = new int[n];

        for(int i = 0; i < indices.length; i++){
            row[indices[i][0]]++;
            col[indices[i][1]]++;
        }

        int oddRows = 0;
        for(int i = 0; i < m; i++){
            if(row[i] % 2 == 1)
                oddRows++;
        }

        int oddCols = 0;
        for(int i = 0; i < n; i++){
            if(col[i] % 2 == 1)
                oddCols++;
        }

        return (oddRows * n) + (oddCols * m) - (2 * oddRows * oddCols);
    }
}

File: Easy/Python/1252. Cells with Odd Values in a Matrix.py
class Solution(object):
    def oddCells(self, m, n, indices):
        row = [0] * m
        col = [0] * n

        for i in range(len(indices)):
            row[indices[i][0]] += 1
            col[indices[i][1]] += 1
        
        oddRows = 0
        for i in range(m):
            if row[i] % 2 == 1:
                oddRows += 1
        
        oddCols = 0
        for i in range(n):
            if col[i] % 2 == 1:
                oddCols += 1
        
        return (oddRows * n) + (oddCols * m) - (2 * oddRows * oddCols)
